upload/src/addons/SV/StandardLib/_output/templates/public/svStandardLib_helper_macros.html

```diff
@@ -0,0 +1,47 @@
+<xf:macro name="date_time_input"
+		  arg-name="date_time"
+		  arg-label=""
+		  arg-explain=""
+		  arg-hint=""
+		  arg-timestampOrDateTimeArr="{{ null }}"
+		  arg-readOnly="{{ false }}"
+		  arg-hours="{{ null }}"
+		  arg-minutes="{{ null }}"
+		  arg-seconds="{{ null }}"
+		  arg-timeZones="{{ null }}">
+	<xf:css src="svStandardLib_helper_macros_date_time_input.less" />
+	
+	<xf:formrow label="{$label}" explain="{$explain}" hint="{$hint}" rowtype="input svStandardLibDateTime">
```

Comment:
If `is_processing` for the add-on is set (ie during upgrades/uninstalls) this code may be cause be template modifications without the resulting listener code being executed.

This mostly impacts the `$timestampOrDateTimeArr`/`$hours`/`$minutes`/`$seconds` values, which don't look to cause errors.

Reply:
Need to split `$timestampOrDateTimeArr` into two fields.

`$timestamp` which is an `int`, and then `$dateTimeArr` which is an array. The reason is that php 7+ will throw errors if you try todo array access on an integer value.

It would be better fo the field to default to an empty value.

Reply:
This should be done.